SYCL/Plugin/interop-level-zero-interop-task-mem.cpp

```diff
@@ -0,0 +1,65 @@
+// REQUIRES: level_zero, level_zero_dev_kit
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %level_zero_options %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// Test for Level Zero interop_task.
+
+#include <CL/sycl.hpp>
+// clang-format off
+#include <level_zero/ze_api.h>
+#include <CL/sycl/backend/level_zero.hpp>
+// clang-format on
+
+#define SIZE 16
+
+class my_selector : public cl::sycl::device_selector {
+public:
+  int operator()(const cl::sycl::device &dev) const override {
+    sycl::backend backend = dev.get_platform().get_backend();
+    if (backend == cl::sycl::backend::level_zero && dev.is_gpu())
+      return 1;
+    else
+      return 0;
+  }
+};
+
+int main() {
+  sycl::queue queue = sycl::queue(my_selector());
+
+  ze_context_handle_t ze_context =
+      queue.get_context().get_native<sycl::backend::level_zero>();
+
+  try {
+    sycl::buffer<uint8_t, 1> buffer(SIZE);
+    sycl::image<2> image(sycl::image_channel_order::rgba,
+                         sycl::image_channel_type::fp32, {SIZE, SIZE});
+
+    queue
+        .submit([&](cl::sycl::handler &cgh) {
+          auto buffer_acc =
+              buffer.get_access<cl::sycl::access::mode::write>(cgh);
```

Comment:
```suggestion
              buffer.get_access<sycl::access::mode::write>(cgh);
```
Please, drop `cl::` namespace everywhere.

I would also add `using namespace sycl;` to simplify the code.